core/src/main/java/io/grpc/internal/ManagedChannelImplBuilder.java

```diff
@@ -104,6 +104,30 @@ public static ManagedChannelBuilder<?> forTarget(String target) {
   private static final long DEFAULT_RETRY_BUFFER_SIZE_IN_BYTES = 1L << 24;  // 16M
   private static final long DEFAULT_PER_RPC_BUFFER_LIMIT_IN_BYTES = 1L << 20; // 1M
 
+  private static final Method GET_CLIENT_INTERCEPTOR_METHOD;
+
+  static {
```

Comment:
Please add a comment explaining why this needs to be in a static block.

Reply:
I'm not sure what that would look like. The main alternative is to make a method and call it `private static final Method GET_CLIENT_INTERCEPTOR_METHOD = getClientInterceptorMethod();`

Reply:
I was just thinking of something like "Look for client interceptor method at class load time to avoid repeatedly calling the class loader", but as I type that it's starting to feel superfluous. 

I'm ok with the way it is, although the 'getClientInterceptorMethod()` is a bit cleaner.

Reply:
More verbose, but I can do that.

Reply:
done